.ci/test.sh

```diff
@@ -18,6 +18,19 @@ else
     CMAKE_OPTS=()
 fi
 
+if [[ $AZURE == "true" ]] && [[ $OS_NAME == "linux" ]] && [[ $TASK == "swig" ]]; then
+    mkdir $BUILD_DIRECTORY/build && cd $BUILD_DIRECTORY/build
+    cmake -DUSE_SWIG=ON "${CMAKE_OPTS[@]}" ..
+    make -j4 || exit -1
+    if [[ $COMPILER == "gcc" ]]; then
+        objdump -T $BUILD_DIRECTORY/lib_lightgbm.so > $BUILD_DIRECTORY/objdump.log || exit -1
+        objdump -T $BUILD_DIRECTORY/lib_lightgbm_swig.so >> $BUILD_DIRECTORY/objdump.log || exit -1
```

Comment:
@imatiach-msft Are there any other dependencies which we may want to check, except those that are already mentioned [here](https://github.com/Microsoft/LightGBM/blob/master/helpers/check_dynamic_dependencies.py)?

Reply:
nope, those are the only two.  you have a script to validate glibc and gomp version?  That's very cool.